bin/hdfcoinc/pycbc_fit_sngls_split_binned

```diff
@@ -0,0 +1,354 @@
+                                 vals_above_thresh, args.stat_fit_threshold)
+
+            # add histogram to plot
+            histcounts, edges = np.histogram(vals_inbin, bins=50)
+            cum_counts = histcounts[::-1].cumsum()[::-1]
+
+            logging.info('setting up fitted values for comparison')
+
+            fitted_cum_counts = len(vals_above_thresh) * \
+                                trstats.cum_fit(args.fit_function, fitrange,
+                                                alpha, args.stat_fit_threshold)
+            fitted_cum_counts_plus = len(vals_above_thresh) * \
+                                      trstats.cum_fit(args.fit_function,
+                                                      fitrange, alpha + sig_alpha,
+                                                      args.stat_fit_threshold)
+            fitted_cum_counts_minus = len(vals_above_thresh) * \
+                                      trstats.cum_fit(args.fit_function, fitrange,
+                                                      alpha - sig_alpha,
+                                                      args.stat_fit_threshold)
+
+
+            # histogram of fitted values
+            ax.semilogy(edges[:-1], cum_counts, linewidth=2,
+                     color=histcolors[i], alpha=0.6)
+            ax.semilogy(fitrange, fitted_cum_counts, "--", color=histcolors[i],
+                     label=r"$\alpha = $%.2f" % alpha)
+            lgd_sub = ax.legend(fontsize='small', framealpha=0.5)
+
+            del vals_inbin, vals_above_thresh
+
+        maxyval = max(maxyval, cum_counts.max())
+        ax.grid()
+
+logging.info('setting up labels and whatnot')
+
+for i in range(args.split_one_nbins):
+    for j in range(args.split_two_nbins):
+        axes[i,j].semilogy([args.stat_fit_threshold, args.stat_fit_threshold],
+                    [1,5*maxyval], 'k', linestyle=':', alpha=0.2)
+
+axes[0,0].set_ylim(1,5*maxyval)
+axes[0,0].set_xlim(minplot, maxplot)
+
+for j in range(args.split_two_nbins):
+    axes[0, j].set_xlabel(args.split_param_two + ': ' + 
+                          (formats[args.split_param_two] + ' to ' +
+                           formats[args.split_param_two]).format(sp_two_bounds.lower()[j],
+                                     sp_two_bounds.upper()[j]), size="large")
+    axes[0, j].xaxis.set_label_position("top")
+    axes[args.split_one_nbins - 1, j].set_xlabel(args.sngl_stat, size="large")
+for i in range(args.split_one_nbins):
+    axes[i, 0].set_ylabel(args.split_param_one + ': ' + 
+                          (formats[args.split_param_one] + ' to ' + 
+                           formats[args.split_param_one]).format(sp_one_bounds.lower()[x],
+                                      sp_one_bounds.upper()[x]) + '\ncumulative number',
+                                      size="large")
+
+axes[0,0].add_artist(lgd)
+
+fig.savefig(args.output_file, bbox_extra_artists=(lgd,), bbox_inches='tight')
```

Comment:
can you look at the existing fit_sngls_binned code https://github.com/gwastro/pycbc/blob/master/bin/hdfcoinc/pycbc_fit_sngls_binned#L394 which uses results.save_fig_with_metadata and see if you can use that (useful for putting plot onto results page)?

Reply:
Done - added in, and in place [here (LIGO login required)](https://ldas-jobs.ligo.caltech.edu/~gdavies/analyses/multiifodev/test_output/background_fits/20190503_065633_H1_a22_split_chieff_eta_triggerhist_newsnr_sgveto_maxdur4_vetoed_singleexp_fitting.png)